src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+  @Test
+  @DisplayName("size() 메소드를 활용해 Set의 크기를 확인한다.")
+  void size() {
+    assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함